packages/mask/src/extension/popups/pages/Wallet/TokenDetail/TrendingChart.tsx

```diff
@@ -17,10 +19,14 @@ interface TrendingChartProps extends HTMLProps<SVGSVGElement> {
 
 export function TrendingChart({ stats, ...props }: TrendingChartProps) {
     const svgRef = useRef<SVGSVGElement>(null)
-    const chartData = useMemo(() => stats.map(([date, price]) => ({ date: new Date(date), value: price })), [stats])
+    const { fiatCurrencyType, fiatCurrencyRate } = useContainer(WalletContext)
+    const chartData = useMemo(
+        () => stats.map(([date, price]) => ({ date: new Date(date), value: price * fiatCurrencyRate })),
```

Comment:
Currency conversion should concern with the loss of precision and overflow issues, try `multipliedBy`.